### DISCUSSION TITLE
Question: Display options

### DISCUSSION THREAD

**Discussion by austwhite** [HASwitchPlate/HASPone#126] [Q&A] [answered]
Hi,  
Just a general question.
Nextion have a couple of different display sizes in their SPI touch range.  Would this project work with Nextion displays of larger sizes, such as 3.1 or 3.5 Inch?

**Comment by aderusha** [comment, marked as answer]
This question comes up a lot so I finally sat down and [wrote up a wiki entry on the topic](https://github.com/HASwitchPlate/HASPone/wiki/Modifying-HASPone-to-work-with-other-Nextion-screens).

Short version: possible, but takes a substantial amount of work.  The project is targeted specifically at 2.4" model NX3224T024 and will remain so.

**Comment by aderusha** [comment]
Moved to discussions so others can join in!
